src/app/ps/semaphore.cpp

```diff
@@ -0,0 +1,99 @@
+#include "semaphore.hpp"
+
+#include <errno.h>
+#include <fcntl.h>
+
+namespace ot {
+
+namespace os {
+
+namespace sem {
+
+#if defined(_WIN32) || defined(WIN32)
+sem_status semaphore_open(std::string const &name, semaphore &sem)
```

Comment:
@mikhail-pridushchenko This is a style issue. We use `Upper Camel Cases` for both class names and method/function names. Could you restyle the names for all files? Thanks!